date.go

```diff
@@ -28,6 +28,8 @@ func dateInZone(fmt string, date interface{}, zone string) string {
 		t = time.Now()
 	case time.Time:
 		t = date
+	case *time.Time:
```

Comment:
If accepting this let me know if I should cover this with a unit test

Reply:
This looks good. Can you please add a test.

Reply:
Sure will do!